scripts/pre-push-check.sh

```
@@ -0,0 +1,149 @@
+# 9. Run UI tests
+print_status "Running UI tests..."
+npm run test:ui
+check_result "UI tests passed"
+echo ""
+
+# 10. Run integration tests (using local database from .env)
+print_status "Running integration tests with local database..."
+
+# Read DATABASE_URL from .env file
+if [ -f ".env" ]; then
+    export $(grep -E '^DATABASE_URL=' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)